articles/native-platforms/ios-objc/05-linking-accounts.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Linking User Profiles
+description: This tutorial will show you how to link different User profiles. Enabling different ways of signing in, into a single profile
```

Comment:
This tutorial will show you how to link different user profiles, allowing different ways of signing in into a single profile.